tests/testthat/test-get_protected.R

```diff
@@ -0,0 +1,30 @@
+test_that("protected nodes can be accessed", {
+  path <- system.file("extdata", "basic-curly.md", package = "tinkr")
+  ex <- tinkr::yarn$new(path, sourcepos = TRUE)
+  # we should have two protected elements right off due to the braces
```

Comment:
as opposed to adding comments, why not create a tinkr object from inline text so we'd see the braces?

Reply:
that's a fair point. I think I was tired when I was doing this.

Reply:
I did this in [4a1228e](https://github.com/ropensci/tinkr/pull/111/commits/4a1228e33e485ad9253e7a5dbf018739ead5b3f1)